packages/webapp/pages/squads/discover/index.tsx

```diff
@@ -4,41 +4,44 @@ import { SquadHorizontalList } from '@dailydotdev/shared/src/components/cards/sq
 import { cloudinary } from '@dailydotdev/shared/src/lib/image';
 import { getLayout } from '../../../components/layouts/FeedLayout';
 import { mainFeedLayoutProps } from '../../../components/layouts/MainFeedPage';
+import { SquadDirectoryLayout } from '../../../../shared/src/components/squads/layout/SquadDirectoryLayout';
 
 function SquadDiscoveryPage(): ReactElement {
   const { data } = useSquadCategories();
   const categories = data?.pages.flatMap((page) => page.categories.edges) ?? [];
 
   return (
-    <div className="relative flex w-full flex-col gap-6 p-5">
-      <div className="absolute inset-0 h-[25rem] w-full bg-gradient-to-t from-accent-cabbage-default from-10% to-background-default" />
-      <SquadHorizontalList
-        key="featured"
-        className="relative"
-        linkToSeeAll="/squads/discover/featured"
-        title={
-          <img
-            className="h-12"
-            src={cloudinary.squads.directory.featured}
-            alt="A text containing the word 'Featured'"
-          />
-        }
-        query={{ isPublic: true, featured: true }}
-      />
-      {categories.map(({ node }) => (
+    <SquadDirectoryLayout>
+      <div className="relative flex w-full flex-col gap-6 p-5">
```

Comment:
Nice! Non-blocking: would be possible to remove this `div` and just pass the classes directly to the `SquadDirectoryLayout`?